whois-api/src/main/java/net/ripe/db/whois/api/rest/mapper/WhoisObjectServerMapper.java

```diff
@@ -28,16 +29,20 @@ public class WhoisObjectServerMapper {
     private final AbuseContactSearch abuseContactSearch;
     private final ManagedAttributeSearch managedAttributeSearch;
 
+    private final List<QueryMessageGenerator> infoMessageValidators;
+
     @Autowired
     public WhoisObjectServerMapper(
             final WhoisObjectMapper whoisObjectMapper,
             final ResourceHolderSearch resourceHolderSearch,
             final AbuseContactSearch abuseContactSearch,
-            final ManagedAttributeSearch managedAttributeSearch) {
+            final ManagedAttributeSearch managedAttributeSearch,
+            final List<QueryMessageGenerator> infoMessageValidators) {
```

Comment:
is it a generator or validator? Use consistency in naming variable and class name

Reply:
I think that we decided to call it generators because we already have Business validator